airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/CloudDeploymentPostgresSourceTest.java

```diff
@@ -73,30 +81,28 @@ private Source source() {
     return PostgresSource.sshWrappedSource(source);
   }
 
-  @Test
-  void testSSlModesDisableAllowPreferWithTunnelIfServerDoesNotSupportSSL() throws Exception {
-    for (final String sslmode : NON_STRICT_SSL_MODES) {
-      final AirbyteConnectionStatus connectionStatus = checkWithTunnel(DB_NO_SSL_WITH_NETWORK, BASTION_NO_SSL, sslmode);
-      assertEquals(AirbyteConnectionStatus.Status.SUCCEEDED, connectionStatus.getStatus());
-    }
+  @ParameterizedTest()
+  @MethodSource("getNonStrictSslModes")
+  void testSSlModesDisableAllowPreferWithTunnelIfServerDoesNotSupportSSL(String sslMode) throws Exception {
```

Comment:
parameterized a bunch of the tests. Trying to show an example of a test that reports some relevant information, but testing several values, and reporting test success/failure for each value separately

Reply:
Thank you for doing this!